minecraft-server/README.md

```diff
@@ -551,6 +551,12 @@ If set to true, players will be set to spectator mode if they die.
 
     docker run -d -e HARDCORE=false
 
+### Snooper
+
+If set to true, server will send snoop data to the minecraft sever.
```

Comment:
Since minecraft itself defaults this to true, perhaps the wording should be flipped. It would also better match your example. So, how about something like

> If set to false, the server will not send data to snoop.minecraft.net

Reply:
Very good point! Modified in the following commit!